cashier-btc.js

```diff
@@ -1,66 +1,65 @@
-/**
- * Cashier-BTC
- * -----------
- * Self-hosted bitcoin payment gateway
- *
- * https://github.com/Overtorment/Cashier-BTC
- *
- **/
-
-process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'
-let express = require('express')
-let morgan = require('morgan')
-let uuid = require('node-uuid')
-
-morgan.token('id', function getId (req) {
-  return req.id
-})
-
-let app = express()
-
-app.use(function (req, res, next) {
-  req.id = uuid.v4()
-  next()
-})
-app.use(morgan(':id :remote-addr - :remote-user [:date[clf]] ":method :url HTTP/:http-version" :status :res[content-length] ":referrer" ":user-agent"'))
-
-app.set('trust proxy', 'loopback')
-
-let bodyParser = require('body-parser')
-let config = require('./config')
-let rp = require('request-promise')
-
-app.use(bodyParser.urlencoded({ extended: false })) // parse application/x-www-form-urlencoded
-app.use(bodyParser.json(null)) // parse application/json
-
-global.btcUsd = 7000 // initial
-global.btcEur = 6000
-
-app.use('/qr', express.static('qr'))
-app.use(require('./controllers/api'))
-app.use(require('./controllers/website'))
-
-let updateExchangeRate = async function (pair) {
-  let json
-  try {
-    json = await rp.get({url: 'https://www.bitstamp.net/api/v2/ticker/' + pair, json: true})
-  } catch (err) {
-    return console.log(err.message)
-  }
-  switch (pair) {
-    case 'btceur': global.btcEur = json.ask; break
-    case 'btcusd': global.btcUsd = json.ask; break
-  }
-}
-
-updateExchangeRate('btcusd').then(updateExchangeRate('btceur'))
-setInterval(() => updateExchangeRate('btcusd').then(updateExchangeRate('btceur')), 5 * 60 * 1000)
-
-require('./smoke-test')
-require('./deploy-design-docs') // checking design docs in Couchdb
-
-let server = app.listen(config.port, function () {
-  console.log('Listening on port %d', config.port)
-})
-
-module.exports = server
+/**

```

Comment:
why is the whole file changed? shouldn't be this way

Reply:
Probably due to my ide's auto formatting issue. let me check.

Reply:
What is the text encoding settings for your ide? mine is utf-8, if yours is iso-8859-1 or anything else it might be the reason why the whole file looks changed.

Reply:
I have once more checked for diffs, and it seems the reason whole file looks has changed is due to whitespace. While checking for diffs, when I check **Hide whitespace changes**  under **Diff settings** the actual changes are shown. Please let me know if you're going to accept his pull request, I'm planning on adding a few more api functions etc, if you're not going to use it, please let me know, I'll maintain my own.